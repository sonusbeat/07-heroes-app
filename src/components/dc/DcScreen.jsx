const ScScreen = () => {
  return ( <h2>Dc Screen</h2> );
}

export default ScScreen;