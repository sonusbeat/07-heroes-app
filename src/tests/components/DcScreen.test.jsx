import { shallow } from "enzyme";
import DcScreen from "../../components/dc/DcScreen";

describe("Puebas en el componente <DcScreen />", () => {
  test("Debe coincidir con el Snapshot", () => {

    const wrapper = shallow( <DcScreen /> );
    expect( wrapper ).toMatchSnapshot();

  });
});