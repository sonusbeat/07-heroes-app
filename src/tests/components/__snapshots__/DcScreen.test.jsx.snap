// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Puebas en el componente <DcScreen /> Debe coincidir con el Snapshot 1`] = `
<section>
  <h2>
    Marvel Comics
  </h2>
  <hr />
  <HeroesList
    publisher="DC Comics"
  />
</section>
`;
